File: App/stadium_reservation/lib/models/Dtos/workdayrequestdto.dart
class Workdayrequestdto {
  String stadiumCenterId;
  int dayOfWeek;
  String beginWorkTime;
  String endWorkTime;

  Workdayrequestdto({
    required this.stadiumCenterId,
    required this.dayOfWeek,
    required this.beginWorkTime,
    required this.endWorkTime,
  });

  Map<String, dynamic> toJson() {
    return {
      'stadiumCenterId': stadiumCenterId,
      'dayOfWeek': dayOfWeek,
      'beginWorkTime': beginWorkTime,
      'endWorkTime': endWorkTime,
    };
  }
}

File: Api/Reservationpitch.Application/DTOs/WorkDayDTOs/Request/WorkDayRequestDto.cs
using Reservationpitch.Domain.Entities;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Reservationpitch.Application.DTOs.WorkDayDTOs.Request
{
    public class WorkDayRequestDto
    {
        public Guid stadiumCenterId { get; set; }
        public DayOfWeek DayOfWeek { get; set; }
        public TimeOnly? BeginWorkTime { get; set; } 
        public TimeOnly? EndWorkTime { get; set; }
    }
}
